src/app.md

```diff
@@ -725,6 +726,119 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given a list of version &strs, finds the latest [semver::Version] that satisfying given [semver::VersionReq] "<= 1.0.0" using [VersionReq::matches].
+
+```rust
+# #[macro_use]
+# extern crate error_chain;
+extern crate semver;
+
+use semver::{Version, VersionReq, AlphaNumeric, Numeric};
+#
+# error_chain! {
+#     foreign_links {
+#         SemVer(semver::ReqParseError);
+#     }
+# }
+
+fn version_latest<'a>(
```

Comment:
I'd probably try to give a more descriptive name here like `find_max_matching_version`